core/vm/contracts.go

```diff
@@ -1432,6 +1433,9 @@ func (c *verifyDoubleSignEvidence) Run(input []byte) ([]byte, error) {
 	}
 
 	// basic check
+	if header1.Number.Cmp(maxUnit256) == 1 || header2.Number.Cmp(maxUnit256) == 1 {
+		return nil, ErrExecutionReverted
+	}
 	if header1.Number.Cmp(header2.Number) != 0 {
 		return nil, ErrExecutionReverted
```

Comment:
`copy(returnBz[52-len(evidenceHeightBz):], evidenceHeightBz)` is not a good code style,  why not start from index 20 directly but `52-len(evidenceHeightBz)`

Reply:
Because we need zero-padding